src/main/features/response/form/models/howMuchDoYouOwe.ts

```diff
@@ -0,0 +1,48 @@
+import { IsDefined, IsPositive } from 'class-validator'
+import { Fractions, IsLessThan, Min } from '@hmcts/cmc-validators'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+import { toNumberOrUndefined } from 'shared/utils/numericUtils'
+import response from 'offer/routes/response'
+
+export class ValidationErrors {
+  static readonly AMOUNT_NOT_VALID: string = 'Enter valid amount'
+  static readonly AMOUNT_LESS_THAN_CLAIMED: string = 'Enter a value less then the claimed amount'
+}
+
+// TODO AMOUNT_LESS_THAN_CLAIMED is a placeholder and has not been approved by PO or content. It was missing in story 3662
+
+export class HowMuchDoYouOwe {
+
+  @IsDefined({ message: DefaultValidationErrors.AMOUNT_REQUIRED })
+  @IsLessThan('totalAmount', { message: ValidationErrors.AMOUNT_LESS_THAN_CLAIMED })
+  @IsPositive({ message: ValidationErrors.AMOUNT_NOT_VALID })
+  @Fractions(0, 2, { message: DefaultValidationErrors.AMOUNT_INVALID_DECIMALS })
+  amount?: number
+
+  totalAmount?: number
```

Comment:
@geek-soft What would you tell about "empty" annotation like `@UsedForOtherFieldsValidation` just to keep it easy to understand why we have this kind of fields. In fact they are not a part of form itself. 

We have more and more fields like this.

Reply:
The longer I think about it the less I like this idea ;)